assets/src/components/v2/triptych/README.md

```diff
@@ -1,5 +1,11 @@
 # Triptych app packaging
 
+## Resizing the PSAs
+Outfront recommends these asset sizes:
+- Images: The boards can take webp, which saves 35% or possibly more. OFM recommends using `sharp`, but you can also use [webp](https://formulae.brew.sh/formula/webp). `cwebp -resize 1080 0 left.png -o left.webp` will resize left.png to the triptych screen size and convert to webp.
```

Comment:
Just double checking, the command is `cwebp`, not just `webp`? (Since the package/formula is called `webp`)

Also, 0 for height? Does the command default to scaling by just one dimension if you set the other to 0?